### ISSUE TITLE
[Bug]: Unable to add custom spans with layer arm64 > 96 and x86 > 213

### ISSUE THREAD

**Issue by janritter** [instana/nodejs#1451]
### Problem Description

When using the newest versions of the Instana Layer in Nodejs with AutoTrace (wrapping the handler), the auto instrumentation part works fine, but trying to access the current span or adding intermediate spans fails (they are null and intermediate spans are not send to Instana).

After creating a Lambda Function with autoTrace and a function URL using the code blow, when executing the Lambda, we can see the following in the logs:

```text
  | 2024-11-18T15:35:33.933Z | INIT_START Runtime Version: nodejs:20.v43 Runtime Version ARN: arn:aws:lambda:eu-central-1::runtime:4a87b8fceb008240eb9bfe7dc3126c2dec6fe75f06c6a4d8977d0267eb5da2bd
  | 2024-11-18T15:35:34.169Z | 2024-11-18T15:35:34.169Z undefined INFO @instana/aws-lambda module version: 3.21.0
  | 2024-11-18T15:35:34.693Z | 2024-11-18T15:35:34.693Z undefined INFO @instana/aws-lambda module version: 4.0.1
  | 2024-11-18T15:35:34.843Z | 2024-11-18T15:35:34.843Z undefined INFO Method warn of [object console] was already wrapped. Unwrapping.
  | 2024-11-18T15:35:34.844Z | 2024-11-18T15:35:34.844Z undefined INFO Method error of [object console] was already wrapped. Unwrapping.
  | 2024-11-18T15:35:34.869Z | EXTENSION Name: instana-lambda-extension State: Ready Events: [INVOKE, SHUTDOWN]
```
  
It looks like the original handler is wrapped in version 3.21.0, while 4.0.1 is used when requiring the package in the code.

### Short, Self Contained Example

Lambda code: 
```javascript
const instana = require('@instana/aws-lambda');

exports.handler = async (event, context) => {
    const span = instana.currentSpan();
    const response = {
        statusCode: 200,
        body: JSON.stringify({
            currentSpanId: span ? span.getSpanId() : null,
        }),
    };
    return response;
};
```

### Node.js Version

Lambda nodejs20.x

### package.json

```shell
-
```

### package-lock.json

```shell
-
```

**Comment by kirrg001** [comment]
Hi @janritter Thanks for your report! We will take a look asap!

**Comment by aryamohanan** [comment]
Hi @janritter 

The fix has been successfully released in [v4.1.0](https://github.com/instana/nodejs/releases/tag/v4.1.0). 

You can now use the following latest layer versions for ARM64 and x86_64 in the region **eu-central-1**:

- x86_64: Layer version 218
- ARM64: Layer version 99

Please let us know if you are still encountering any problems.

**Comment by janritter** [comment]
@aryamohanan thanks for the quick release, I can confirm that it now works on our end
